tests/queries/0_stateless/01213_alter_rename_with_default_zookeeper_long.sql

```diff
@@ -38,11 +38,11 @@ CREATE TABLE table_rename_with_ttl
   date1 Date,
   date2 Date,
   value1 String,
-  value2 String TTL date1 + INTERVAL 10000 MONTH
```

Comment:
@alesapin Could you explain why we choose `10000` month for TTL? It leads to implementation defined date overflow.